app/models/bin.rb

```diff
@@ -1,4 +1,8 @@
 class Bin < ActiveRecord::Base
   validates :payload, presence: true
```

Comment:
We also need a validation for the maximum size of the payload

Reply:
I added a max size of 10k